azure-iot-common/tests/test_asyncio_compat.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import asyncio
+import sys
+from azure.iot.common import asyncio_compat
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+def dummy_coroutine():
+    async def coro():
+        return
+
+    return coro
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="Requires Python 3.7+")
+async def test_get_running_loop(mocker, event_loop):
+    spy_get_running_loop = mocker.spy(asyncio, "get_running_loop")
+    result = asyncio_compat.get_running_loop()
+    assert result == event_loop
+    assert spy_get_running_loop.call_count == 1
+    assert spy_get_running_loop.call_args == mocker.call()
+
+
+@pytest.mark.skipif(sys.version_info >= (3, 7), reason="Requires Python 3.6 or below")
+async def test_get_running_loop_compat(mocker, event_loop):
+    spy_get_event_loop = mocker.spy(asyncio, "_get_running_loop")
+    result = asyncio_compat.get_running_loop()
+    assert result == event_loop
+    assert spy_get_event_loop.call_count == 1
+    assert spy_get_event_loop.call_args == mocker.call()
+
+
+@pytest.mark.skipif(sys.version_info >= (3, 7), reason="Requires Python 3.6 or below")
+async def test_get_running_loop_compat_no_running_loop(mocker):
+    mocker.patch.object(asyncio, "_get_running_loop", return_value=None)
+    with pytest.raises(RuntimeError, message="Expecting Runtime Error"):
+        asyncio_compat.get_running_loop()
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="Requires Python 3.7+")
+async def test_create_task(mocker, dummy_coroutine):
+    spy_create_task = mocker.spy(asyncio, "create_task")
+    coro_obj = dummy_coroutine()
```

Comment:
>dummy_coroutine [](start = 15, length = 15)

the word "dummy" will eventually get flagged by one of the code scanners as being potentially offensive.  Use "fake" instead.